### DISCUSSION TITLE
Songs don't play after being copied to an iPod Shuffle (2nd gen)

### DISCUSSION THREAD

**Discussion by RaptaG** [strawberrymusicplayer/strawberry#1267] [Q&A] [answered]
After I select a song (in MP3 format) and click `Copy to Device` -> `IPOD` -> `OK` the song is transferred in the iPod flawlessly. When trying to play it though, it does not play. Therefore, I'm forced to use [a workaround](https://shuffle-db.sourceforge.net), which I really want to stop using since it is a very bad, hacky and old implementation for that job.

I remember reading somewhere that it is an issue with [`libgpod`](https://github.com/fadingred/libgpod).

Even so, can it be fixed in Strawberry? I can test any potential patches if needed, just lemme know.

Thanks in advance!

**Comment by jonaski** [comment, marked as answer]
libgpod is no longer maintained and it does not work for all iPods.
